### PR TITLE
9. GET /api/articles 

### PR DESCRIPTION
**Pull request by deasar-js**
GET /api/articles complete, tests for properties by loop

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -28,6 +28,38 @@ describe("GET /api/topics", () => {
...
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(12);
+        articles.forEach((article) => {
+          expect(
+            typeof article.author &&
```
Comment (douglashellowell) [inline __tests__/index.test.js:42]: This code isn't doing what you think it's doing i'm afraid!

you would need to write it like this

```js
expect(
    typeof article.author === 'string' &&
    typeof article.title === 'string' &&
    typeof article.topic === 'string' &&
    // ... etc    
    )
```

a neater way would be to use `expect.any(String)`

**Review by douglashellowell — CHANGES_REQUESTED** [review]
Change your tests so they're testing data types more accurately!

Other than that looking great!

File: __tests__/index.test.js (hunk 1 above)
Comment (deasar-js) [reply __tests__/index.test.js:42]: okay, thanks for the help will work on the changes now

**Comment by deasar-js** [comment]
Is this correct?

**Comment by rogersop** [comment]
> 

Looks fine now @deasar-js 👍

**Pull request merged**